docs/content/handover-between-server-workers.md

```diff
@@ -0,0 +1,49 @@
+
+# Actor property handover between SpatialOS servers
+## Overview
+
+In Unreal’s native client-server architecture, your game server holds the canonical state of the whole game world. As there is a single game server, there are Actor properties that the server doesn’t need to share with any other server or clients. These properties only need to exist in the game server’s local process space. 
+
+In SpatialOS games the work of the server is spread across several servers (known as “server-workers” in SpatialOS). (Note that in SpatialOS, game clients are “client-workers” - there’s more information on [workers](https://docs.improbable.io/reference/13.1/shared/concepts/workers) in the SpatialOS documentation.)
+
+As Unreal expects there to be only one server, rather than several servers, the Unreal GDK has a custom solution to take advantage of the SpatialOS distributed server architecture. This involves a handover of responsibility for an Actor and its properties between server-workers. (Actors approximate to “entities” in SpatialOS, so we refer to them as “entities” when we are talking about what happens to them in SpatialOS - handily, “properties” in an entity’s components in SpatialOS map to replicated Actor properties. You can find out more about [entities, components and properties](https://docs.improbable.io/reference/13.1/shared/concepts/entities) in the SpatialOS documentation.)
+
+Server-workers have [authority](https://docs.improbable.io/reference/13.1/shared/design/understanding-access#understanding-read-and-write-access-authority) (SpatialOS documentation) over entities, meaning that they are responsible for properties of an entity. Only one server-worker has authority over the properties of an entity at a time. In order to [load balance](https://docs.improbable.io/reference/13.1/shared/glossary#load-balancing)(SpatialOS documentation) between server-workers, each server-worker has only a certain area of authority, so each server-worker has a boundary. 
+This means that, at the boundary between server-worker 1 and server-worker 2,  server-worker 1 needs to transfer authority of entity properties to server-worker 2 so that server-worker 2 can seamlessly continue to simulate the entity exactly where server-worker 1 stopped. (See SpatialOS documentation on [Authority change](https://docs.improbable.io/reference/13.1/shared/design/operations#authoritychange).)
+
+
+Note that server-worker authority over properties is different to server-worker interest in properties. See SpatialOS documentation on [worker interest](https://docs.improbable.io/reference/13.1/shared/glossary#interest).
+
+## How to facilitate Actor handover
+
+To facilitate an Actor’s property handover between server-workers, follow the instructions below:
+1.  In the Actor’s class, mark the property field with a UPROPERTY `Handover` tag, as shown in the example below.
+
+```
```

Comment:
Does this code align properly in preview, as it is part of a list it should probably be indented